android/src/main/java/sqip/flutter/SquareInAppPaymentsFlutterPlugin.java

```diff
@@ -48,7 +48,8 @@ public void onMethodCall(MethodCall call, final Result result) {
       InAppPaymentsSdk.INSTANCE.setSquareApplicationId(applicationId);
       result.success(null);
     } else if (call.method.equals("startCardEntryFlow")) {
-      cardEntryModule.startCardEntryFlow(result);
+      boolean collectPostalCode = call.argument("collectPostalCode");
```

Comment:
Could we provide a way for the developer to optionally add `collectPostalCode` and if  `call.argument("collectPostalCode");` is not provided then call `cardEntryModule.startCardEntryFlow(result, false);`?